Frontend/CO.CDP.OrganisationApp/Pages/Shared/_Layout.cshtml

```diff
@@ -130,10 +130,10 @@
         <div class="govuk-phase-banner">
             <p class="govuk-phase-banner__content">
                 <strong class="govuk-tag govuk-phase-banner__content__tag">
-                    Prototype
+                    Beta
                 </strong>
                 <span class="govuk-phase-banner__text">
-                    This is a prototype of a service in development.
+                    <a class="govuk-link" target="_blank" href="/provide-feedback-and-contact/?context=feedback">Contact the Find a Tender team</a> if you have feedback, questions or suggestions.
```

Comment:
Links that open in a new tab need to be marked as such. As in the following example from https://design-system.service.gov.uk/components/phase-banner/#add-a-feedback-link

<img width="503" alt="{3DF1029B-C4ED-4D08-9671-45BDB7D2D0AD}" src="https://github.com/user-attachments/assets/470b0d63-e6d4-464a-a21a-35e7fac682aa">

Reply:
Done